scipy/spatial/transform/tests/test_rotation.py

```diff
@@ -1821,6 +1991,7 @@ def test_slerp(xp):
     assert_equal(len(interp_rots), len(times))
 
 
+@make_xp_test_case()
```

Comment:
This doesn't do anything and makes no sense.
Did you try wrapping marking `Slerp.__init__` with `@xp_capabilities`?
(a better solution would be to add classes support to `xp_capabilities` so that we can benefit from auto-documentation, but that's out of scope for this PR)

Reply:
Yes. Now using `Slerp.__init__` explicitly. And yes, class support would be nice to have in these kinds of situations